united.py

```diff
@@ -1,25 +1,51 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+import calendar
 import csv
 from datetime import datetime
 import os
 import argparse
+from wsgiref import headers
+from xml.etree import ElementTree
 
-from flask.ext.sqlalchemy import SQLAlchemy
+import re
+from flask_sqlalchemy import SQLAlchemy
 from sqlalchemy import and_
 
 from models import Marker, MARKER_TYPE_ACCIDENT
 from utilities import init_flask
 import importmail
 
+import urllib2
+
+from xml.dom import minidom
+from xml.dom.minidom import Document
+
+import math
+
+import requests
+import logging
+
+
+import Cookie
 
 ############################################################################################
 # United.py is responsible for the parsing and deployment of "united hatzala" data to the DB
 ############################################################################################
 
 PROVIDER_CODE = 2
 
+def retrive_ims_xml():
+
+    logging.basicConfig(level=logging.DEBUG)
+    s = requests.session()
+    r = s.get('http://www.ims.gov.il/ims/PublicXML/observ.xml')
+    xml_doc = minidom.parseString(r.text)
+    collection = xml_doc.documentElement
+    return collection
+
+collection = retrive_ims_xml()
```

Comment:
This line happens outside any function, so also if the file is imported. Better put it in one of the functions and then pass `collection`.